benchmarks/benchmark_run.sh

```diff
@@ -0,0 +1,168 @@
+#! /bin/bash
+set -e
+
+OMP_NUM_THREADS=24 # need to be modified based on cpus_per_worker
+VALUE_INF=2000
+choice=${1}
+
+get_peak_throughpt(){
```

Comment:
Could we print out the key paramaters before the run so that people can know what is running ?